packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -383,6 +382,7 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate,
   }
 
   updateItemSize(key: Key, size: Size) {
+    if (size.height === 0) {return false;}
```

Comment:
It's not clear if it is a safe change. Probably the height of a list item is 0 only if the `display:none` set. 
On the other hand if somehow the item size is small enough (say 1), react will throw the same exception even if the ListView is visible. This is because after render the size is calculated (1 in this case) and a re-render will add one more item to the visible item's list, which will notice it's size and render again... If the collection is large enough react will treat it as an infinite recursion.

Reply:
The page does not crash if padding is set to 1 on ListLayout, but this is not configurable. Can be verified by setting the padding in ListView.tsx.